region/lowernorfair/east.json

```diff
@@ -970,14 +1006,75 @@
           "to": [
             {
               "id": 2,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    {"or": [
+                      "canUseMorphBombs",
+                      {"and": [
+                        "canUsePowerBombs",
+                        {"cost": [
+                          {"ammo": [{
+                            "type": "PowerBomb",
+                            "count": 2
+                          }]}
+                        ]}
+                      ]}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canHeatRun",
+                    "canUsePowerBombs",
```

Comment:
Same question as 2 to 3

Reply:
I'll add it

Reply:
The bottom door requires a PB only once, though. If it's in any way possible to open that door, come out, and farm PBs, then that PB cost can likely be ignored, unlike actual passage.

Reply:
Code looks better without the divide between heat and proof anyway ^^